flink-ml-benchmark/README.md

```diff
@@ -194,3 +189,9 @@ stage. The stage is benchmarked against 10000 randomly generated vectors.
   }
 }
 ```
+
+### Benchmark Results Visualization
+
+`benchmark-results-visualize.py` is provided as a helper script to visualize
```

Comment:
Could we provide commands that users can copy and paste to visualize the `results.json` in this quickstart?